OpenUtau.Core/Ustx/UTrack.cs

```diff
@@ -89,8 +88,9 @@ public class UTrack {
         [YamlIgnore] public string SingerName => Singer != null ? Singer.DisplayName : "[No Singer]";
         [YamlIgnore] public int TrackNo { set; get; }
         public string TrackName { get; set; } = "New Track";
-        public bool Mute { set; get; }
-        public bool Solo { set; get; }
+        public bool ApparentMute { set; get; }
```

Comment:
please annotate this field with `[YamlIgnore]` to be not saved in ustx. And I think `Muted` will be a better name.

Reply:
ok, I fixed it.